lib/sr.py

```diff
@@ -166,7 +166,7 @@ def get_type(self) -> str:
             self._type = self.pool.master.xe("sr-param-get", {"uuid": self.uuid, "param-name": "type"})
         return self._type
 
-    def create_vdi(self, name_label: str, virtual_size: int = 64, image_format: Optional[str] = None) -> VDI:
+    def create_vdi(self, name_label: str, virtual_size: int | str = '1GiB', image_format: Optional[str] = None) -> VDI:
```

Comment:
I'm not a fan of type unions except in special cases. Here, I think we should define an alias because I'm afraid we'll have to specify the union in several places in the tests in the future like resize tests.

For example:
```python
Size = int | str

...

def create_vdi(self, name_label: str, virtual_size: Size = '1GiB', image_format: Optional[str] = None) -> VDI:
```

Otherwise it's fine to use: 2**30 directly.